FhirToDataLake/NuGet.Config

```diff
@@ -4,5 +4,6 @@
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="FhirAnalyticsPublic" value="https://microsofthealthoss.pkgs.visualstudio.com/FhirAnalytics/_packaging/FhirAnalyticsPublic/nuget/v3/index.json" />
+    <add key="Microsoft Health OSS" value="https://microsofthealthoss.pkgs.visualstudio.com/FhirServer/_packaging/Public/nuget/v3/index.json" />
```

Comment:
Can we keep the same format without blank?

Reply:
The spaces in NuGet source key? I follow up the key in fhir-server project [nuget.config](https://github.com/microsoft/fhir-server/blob/main/nuget.config)